paasta_tools/instance/kubernetes.py

```diff
@@ -577,31 +633,54 @@ def get_pod_status(pod: V1Pod, backends: Optional[Set[str]],) -> Dict[str, Any]:
 
     return {
         "name": pod.metadata.name,
+        "service": pod.metadata.labels.get("paasta.yelp.com/service", "Unknown"),
+        "instance": pod.metadata.labels.get("paasta.yelp.com/instance", "Unknown"),
```

Comment:
I used these later on the client side for generating a pod's table entry in `create_replica_table` (e.g. when we generate the tips that include `paasta logs` invocations), since we only passed in the pod result.

Is the preference to just pass those args down from `print_kubernetes_status` through to `create_replica_table` rather than add this to the API stucture?

Reply:
Oh hmm.  I guess I would prefer that we just passed them down if that's not hard, but if that involves threading them through a long chain of function calls I am happy to do this instead.